reviews/layout/helpers.py

```diff
@@ -10,9 +10,14 @@
 from rich.table import Table
 from rich.tree import Tree
 
+from reviews.config import get_label_colour_map
 from reviews.datasource import PullRequest
 
 
+def _colourise_label(label: str) -> str:
+    return get_label_colour_map().get(label.lower(), "[white]") + label
```

Comment:
I'm happy with defaulting to white if the colour choice is invalid. I'll create another issue to supply a command to validate colour configuration.